Documentation/FontEditor/UsingFontEditor.md

```diff
@@ -0,0 +1,123 @@
+
+### Launch another instance
+![](FontEditor_Launching_second_instance.png)
+
+Figure above shows where you can find FontEditor from inside the SerenityOS desktop. It is joined together with other similarly amazing applications like GML Playground, Hack Studio, Hex Editor, Inspector, Keyboard Mapper, Profiler and Theme Editor. Each of the apps by themselves are worthy of their own self-contained tutorial which we hope to soon see materialize. These apps are all native to SerenityOS as they are for SerenityOS, by SerenityOS and of SerenityOS. Python was included simply because this is the `Development` section of Serenity menu. Python is part of the ` Ports ` package and is actively being managed and maintained by [@linusg](https://github.com/linusg).
+
+### Side by side
+![](FontEditor_Twins_sidebyside.png)
+
+Having another instance of FontEditor can help boost productivity. This is most noticeable when one is just starting out on using FontEditor and is trying out how to best make use of the application. Don't be afraid to experiment, let your inner font master slowly shine through. It is best to maintain the feeling of having fun while silently grinning from ear to ear as you steadily create form and personality into your font. Don't be afraid to start again, if you must. As with the entire workflow, being aware when to start, when to stop and when to reset is crucial. With FontEditor to assist you, starting from scratch is no longer an arduous process.
+
+### Save font
+![](FontEditor_Save_font_as.png)
+
+Save your font by following the recommended ` FontName + FontStyle + FontPresentationSize + .font ` naming convention.
+
+### Continue Editing
+![](FontEditor_Continue_editing_current_font.png)
+
+The figure above shows some glyphs already created. Those with sharp eyes can immediately see that the glyphs for M and W, O and Q came from the same base. W was copied over from M and was flipped horizontally. Q was copied over from O and descender was added to achieve the final glyph. Continue editing your font by adding more glyphs into it. Take your time in creating the font, only you know the reason why the glyph is formed the way it is. You know why the height and the width is so, why the curvature is just so. Aside from achieving balance, maintain the spirit of fun in making your font. The more glyphs are added, the easier it is to get the general feel and character of the font. Remember to always save and to save often.
+
+### fonts.serenityos.net
+![](Fonts_SerenityOS_dot_Net.png)
+
+It is highly recommended to use our own [fonts portal](https://fonts.serenityos.net) as one of your primary resource for glyph and font information. The others are [Unicode charts [unicode.org]](https://www.unicode.org/charts/) and our [wiki](https://wiki.serenityos.net).
+
+### Search glyph
+![](Fonts_SerenityOS_dot_Net_search_result.png)
+
+Figure above depicts a search session on [fonts.serenityos.net](https://fonts.serenityos.net) portal.
```

Comment:
The figure above depicts a search session on the fonts.serenityos.net portal.

Reply:
Updated as such.